src/ssh.ext.c

```diff
@@ -19,13 +21,16 @@ void sshQ___ext_init__() {
         acton_gc_strdup,
         acton_gc_strndup);
     int r = ssh_init();
-    printf("SSH extension initialized %d\n", r);
+    if (r != SSH_OK)
+      printf("SSH init failed (%d)\n", r);
+    else
+      printf("SSH extension successfully initialized (retval: %d)\n", r);
 }
 
 B_str sshQ_version () {
-    if (LIBSSH_VERSION_MINOR != 11)
-        return to$str("invalid");
-    return to$str("0.11.0");
+    if (LIBSSH_VERSION_MAJOR != 0 || LIBSSH_VERSION_MINOR != 11 || LIBSSH_VERSION_MICRO != 0)
```

Comment:
Is this code reachable? I mean, we define which ssh version is used in the build.act.json, right?

Reply:
it is reachable. this is the output if you run the ssh-client (notice the line after `SSH extension successfully initialized (retval: 0)`):

```bash
./out/bin/ssh
SSH extension successfully initialized (retval: 0)
libssh 0.11.0 supported

        sshQ_ClientD__initG_local session:              0x7d9662992800
        sshQ_ClientD__initG_local self->session:        0x7d9662992800
Connecting to SSH server 'localhost'
 11:16:50 up  4:47,  4 users,  load average: 0.23, 0.32, 0.35
SSH Client connected
        sshQ_ChannelD__initG_local channel: 0x7d9662982bd0
        sshQ_ChannelD__initG_local self->_ssh_channel:  0x7d9662982bd0
SSH Channel created
```

if one of the checks is change, for example to `LIBSSH_VERSION_MINOR != 123`

```bash
./out/bin/ssh
SSH extension successfully initialized (retval: 0)
unsupported version

        sshQ_ClientD__initG_local session:              0x7999734c9800
        sshQ_ClientD__initG_local self->session:        0x7999734c9800
Connecting to SSH server 'localhost'
 11:16:56 up  4:47,  4 users,  load average: 0.27, 0.32, 0.35
SSH Client connected
        sshQ_ChannelD__initG_local channel: 0x7999734b9bd0
        sshQ_ChannelD__initG_local self->_ssh_channel:  0x7999734b9bd0
SSH Channel created
```

Reply:
Ah, now i get it. This is just dummy code that can be removed. It was there just to get a better understanding of Acton. Adding more version values checks was unnecessary and just confusing.

Will drop that `version()` related code